arbor/fvm_layout.hpp

```diff
@@ -19,15 +19,52 @@
 namespace arb {
 
 // CV geometry as determined by per-cell CV boundary points.
+//
+// Details of CV cable representation:
+//
+//   * The extent of the cables of a control volume corresponds to the
+//     closure of the control volume on the morphology tree.
+//
+//   * Every fork in the morphology tree 'belongs' to exactly
+//     one CV. A fork belongs to a CV if and only if there is
+//     a cable in the CV for each branch of the fork, that
+//     includes the fork point.
+//
+//   * If a CV has more than one cable covering a fork point, then
+//     that fork point must belong to that CV.
+//
+// These requirements are a consequence of the CV geometry being determined
+// by a collection of CV boundarary locations.
+
+namespace cv_prefer {
```

Comment:
Why not use a scoped enum?

Reply:
Mainly because we can then `use namespace cv_prefer`.

Reply:
At least until we get the ability to bring class enum names into scope with `using`, this seems a more convenient way of doing things — we have the option of bringing them in or not, and it's still not polluting the current namespace.

Reply:
Not a pattern I have seen before, and feels like a bit of a long winded way to avoid the verbosity of a scoped enum. When reading the code I see the `using namespace` and these symbols and think "probably an enum in a namespace, maybe?"

Reply:
I disagree with it being long-winded; it's one or two lines of code, that can save egregiously verbose enum value names later. I would like to keep this usage pattern, pending something like [P1099r5](http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2019/p1099r5.html) getting in.

Reply:
Not long winded, but elaborate. But I understand that you have a strange aversion to scoped enums.